lib/Echidna/Test.hs

```diff
@@ -66,11 +68,20 @@ checkETest t = do
         Nothing -> False
   res <- case t of
     -- If our test is a regular user-defined test, we exec it and check the result
-    Left  (f, a) -> execTx (Tx (SolCall (f, [])) (s a) a g 0 0 (0, 0)) >> gets (p f . getter)
+    Left  (f, a) -> do
+      sd <- hasSelfdestructed a
+      _  <- execTx (Tx (SolCall (f, [])) (s a) a g 0 0 (0, 0)) 
```

Comment:
we can use a `basicTx` here (please merge the latest `master`)

Reply:
Fixed.